### PR TITLE
Stacksmith merger

### PR DESCRIPTION
**Pull request by sameersbn**
- use `gcr.io/stacksmith-images/ubuntu:14.04-r06` base image
- use harpoon redis module
- volume unification at `/bitnami/redis`
- start redis-server with `harpoon start`
- logging to stdout via harpoon

### CODE REVIEW

**Comment by sameersbn** [comment]
ok to test

**Comment by sameersbn** [comment]
okay to test

**Comment by sameersbn** [comment]
retest this please

**Comment by sameersbn** [comment]
retest this please

**Comment by sameersbn** [comment]
Ready to merge
/cc @prydonius

File: README.md [hunk 1]
```diff
@@ -40,82 +36,64 @@ docker pull bitnami/redis:[TAG]
...
+If you have already started using your database, follow the steps on [backing up](#backing-up-your-container) and [restoring](#restoring-a-backup) to pull the data from your running container down to your host.
 
-The Redis image exposes a volume at `/bitnami/redis/data`, you can mount a directory from your
-host to serve as the data store. If the directory you mount is empty, the database will be
-initialized.
+The image exposes a volume at `/bitnami/redis` for the Redis data and configurations. For persistence you can mount a directory at this location from your host. If the mounted directory is empty, it will be initialized on the first run.
 
 ```bash
-docker run -v /path/to/data:/bitnami/redis/data bitnami/redis
+docker run -v /path/to/redis:/bitnami/redis bitnami/redis:latest
```
Comment (prydonius) [inline README.md]: `/path/to/redis` is a bit confusing, I think we can call it data even though it's data and conf now. Open to other ideas too.

File: README.md [hunk 2]
```diff
@@ -167,85 +143,72 @@ redis:
 
 A [replication](http://redis.io/topics/replication) cluster can easily be setup with the Bitnami Redis Docker Image using the following environment variables:
 
- - `REDIS_REPLICATION_MODE`: Replication mode. Possible values `master`/`slave` (default: none).
- - `REDIS_MASTER_HOST`: Hostname/IP of replication master (parameter available only on slave).
- - `REDIS_MASTER_PORT`: Port of replication master, defaults to `6379` (parameter available only on slave).
- - `REDIS_MASTER_PASSWORD`: Master auth password used by slave to authenticate with the master (default: none).
+ - `REDIS_REPLICATION_MODE`: The replication mode. Possible values `master`/`slave`. No defaults.
+ - `REDIS_MASTER_HOST`: Hostname/IP of replication master (slave parameter). No defaults.
+ - `MARIABD_MASTER_PORT`: Server port of the replication master (slave parameter). Defaults to `6379`.
```
Comment (prydonius) [inline README.md]: MariaDB? :P

Comment (sameersbn) [inline README.md]: 😛

File: README.md (hunk 1 above)
Comment (sameersbn) [inline README.md]: How aboult `/path/to/redis-persistence`?

Comment (prydonius) [inline README.md]: it's verbose, but makes it's pretty clear, so I think that'll work :)

**Comment by sameersbn** [comment]
updated.

**Comment by sameersbn** [comment]
updated

**Pull request merged**